experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -104,6 +114,10 @@ end
   return cartan_bilinear_form(cartan_matrix(R); check=false)
 end
 
+@attr QQMatrix function bilinear_form_QQ(R::RootSystem)
```

Comment:
I think something like
```suggestion
@attr QQMatrix function bilinear_form(::QQField, R::RootSystem)
```
would be more OSCAR-y?

Reply:
Yeah, but this does not allow `@attr` to work. Furthermore, this is just supposed to be internal

Reply:
I renamed it now to `_bilinear_form_QQ` to make it more visible that its internal